### DISCUSSION TITLE
AS5600 Position Sensor

### DISCUSSION THREAD

**Discussion by TrDA-hab** [arendst/Tasmota#10562] [Show and tell]
### Note!
**Now (01/15/2021) Tasmota has no support for the AS5600 sensor. Maybe support for the AS5600 will be added at a later date.**

![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/AS5600/AS5600-4.jpg)

## More information:
 - [About as5600](https://ams.com/as5600)
 - [Datasheet as5600](https://ams.com/documents/20143/36005/AS5600_DS000365_5-00.pdf)
 - [Github as5600](https://github.com/kanestoboi/AS5600)
 - [Thingiverse as5600](https://www.thingiverse.com/thing:4007896)
 - [Seeedstudioe as5600](https://wiki.seeedstudio.com/Grove-12-bit-Magnetic-Rotary-Position-Sensor-AS5600/)
 - [as5600 + Stepper motor](https://hackaday.io/project/164193-as5600-stepper-motor-encoder)
 - [as5600 + Servo motor](https://hackaday.io/project/17079-mocoder-magnetic-encoder)
 - [as5600 + DC motor](https://www.thingiverse.com/thing:4030416)
 - [Test as5600](https://www.thingiverse.com/thing:4656550)

## BOM:
 - [as5600 (aliexpress)](https://aliexpress.ru/item/4000551682522.html)
 - [AliexpressGood magnet (aliexpress)](https://aliexpress.ru/item/32750723146.html)

## My practice:
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/AS5600/AS5600%2BA4988-0.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/AS5600/20200507_155953.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/AS5600/20200507_181120.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/AS5600/20200507_181514.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/AS5600/20200507_185034.jpg)

Best regards  
TrDA

**Comment by vic42** [comment]
Beautiful!
